### PR TITLE
Add a warning with a redirect to the old docs

### PR DESCRIPTION
**Pull request by indigo423**
To help our people stranded on the new docs .com page.

### CODE REVIEW

**Review by RangerRick — APPROVED** [review]

File: start-page/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Welcome to the OpenNMS Documentation
 
+IMPORTANT: We are migrating our docs to the https://antora.org[Antora framework].
+           This place here is work in progress and not complete.
```
Comment (Bonrob2) [inline start-page/modules/ROOT/pages/index.adoc]:
```suggestion
           This location is work in progress and not complete.
```

File: start-page/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+           If you want the official OpenNMS documentation, please go to https://docs.opennms.org.
```
Comment (Bonrob2) [inline start-page/modules/ROOT/pages/index.adoc]:
```suggestion
           For the official OpenNMS documentation, please go to https://docs.opennms.org.
```

File: start-page/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+           Feel free to get in contact with us in our https://chat.opennms.com/opennms/channels/wtd[Chat] if you have questions.
```
Comment (Bonrob2) [inline start-page/modules/ROOT/pages/index.adoc]:
```suggestion
           Feel free to contact us in our https://chat.opennms.com/opennms/channels/wtd[Chat] if you have questions.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor change. Thanks for doing this Ronny!

**Pull request merged**